tests/AtelierTomato.Markov.Model.Test/BookObjectOIDTest.cs

```diff
@@ -127,5 +127,23 @@ public void BookParseSentenceNotIntTest()
 			Action act = () => BookObjectOID.Parse("Book:_:Alice in Wonderland:Through the Looking Glass:1:3:Wrong");
 			act.Should().Throw<ArgumentException>().WithMessage("The part of the BookObjectOID corresponding to the sentence was not able to be parsed into an integer value. (Parameter 'OID')");
 		}
+		[Fact]
+		public void IncrementBookOID()
+		{
+			BookObjectOID oid = BookObjectOID.ForParagraph("_", "_", "_", "_", 1);
+			IEnumerable<string> strings = ["one", "two", "three"];
+
+			List<Sentence> result = strings.Select((text, index) =>

```

Comment:
copying the place where you know it's used is nice and all but a simple test where you just do one would be more proper for how coverage is usually done. also this doesn't check for non-mutation of the original oid